deploy/deploy-wmagent.sh

```diff
@@ -131,7 +131,7 @@ check_certs()
     echo -e "\n  ... nope, trying to copy them from another node, you might be prompted for the cmst1 password."
     set -e
     if [[ "$IAM" == cmst1 ]]; then
-      scp cmst1@vocms0250:/data/certs/* /data/certs/
+      scp cmst1@vocms0255:/data/certs/* /data/certs/
```

Comment:
Just a comment here, No need for a change in this PR. We'll have to think of a better way of distributing this instead of hardcoding a machine name in the deployment script. But is not worth the effort before we figure out how we should proceed with the token based authentication.